tests/acceptance/10_files/14_immutable/00_immutable.cf

```diff
@@ -0,0 +1,64 @@
+body common control
+{
+  bundlesequence  => { "init", "test", "check", "clean" };
```

Comment:
doesn't the dcs framework handle this sequence?

There is "destroy" which you could use instead of clean. FWIW.

https://github.com/cfengine/core/blob/master/tests/acceptance/dcs.cf.sub#L208-L221

Reply:
I think it's nice to be able to execute a stand-alone file. A bit unnecessary to include the dcs framework just to get a predefined bundle sequence. But I'm open to include it, if you think that's better. Maybe there are some other benefits that I'm not aware of.

Reply:
dcs does some other things with test metadata IIRC. It's not critical to have and it does add some overhead. Probably the nice things dcs does should be elsewhere.